src/logging.cc

```diff
@@ -786,7 +794,7 @@ inline void LogDestination::LogToSinks(LogSeverity severity,
                                        size_t message_len) {
   ReaderMutexLock l(&sink_mutex_);
   if (sinks_) {
-    for (int i = sinks_->size() - 1; i >= 0; i--) {
+    for (ptrdiff_t i = sinks_->size() - 1; i >= 0; i--) {
```

Comment:
Just thought about this. You can write

```c++
for (size_t i = sinks_->size() - 1; i < sinks_->size(); --i)
    // ...
```
If `i` overflows, the loop will end. This is also safe, since unsigned integers are guaranteed to wrap.